src/index.html

```diff
@@ -22,6 +22,394 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="#"
+          class="header__logo-link"
+        >
+          <img
+            src="images/logo.png"
+            alt="header__logo"
+            class="logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item is-active"
+              >
```

Comment:
The 'is-active' class suggests that it's a modifier, but it does not follow the BEM naming convention. Use 'nav__item--is-active' instead.